targets/extension

```diff
@@ -29,12 +29,33 @@ if [ "$XMETHOD" = 'x11' ]; then
 fi
 
 # Use croutonurlhandler as a fallback URL handler
-handler='/usr/local/bin/croutonurlhandler'
-for link in x-www-browser gnome-www-browser; do
-    if ! update-alternatives --query "$link" | grep -q "$handler"; then
-        update-alternatives --install "/usr/bin/$link" "$link" "$handler" 15
-    fi
-done
+if [ "${DISTROAKA:-"$DISTRO"}" = 'debian' ]; then
+    handler='/usr/local/bin/croutonurlhandler'
+    for link in x-www-browser gnome-www-browser; do
+        if ! update-alternatives --query "$link" | grep -q "$handler"; then
+            update-alternatives --install "/usr/bin/$link" "$link" "$handler" 15
+        fi
+    done
+elif [ "${DISTROAKA:-"$DISTRO"}" = 'arch' ]; then
+    install arch=desktop-file-utils,
+
+    cat > '/usr/local/share/applications/crouton.desktop' <<END
+[Desktop Entry]
+Encoding=UTF-8
+Type=Application
+Name=Crouton URL handler
+GenericName=Web Browser
+Comment=Open URL in Chromium OS via crouton extension
+Exec=/usr/local/bin/croutonurlhandler %U
+Icon=crouton
```

Comment:
No (as far as I remember). Might be worth installing the extension icon and use that.